### PR TITLE
chore: copy most settings from agrc/open-sgid

### PR DESCRIPTION
**Pull request by stdavis**
I'd like to try this out, but it's looking good to me so far...

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*.{cs, py}]
```
Comment (steveoh) [inline .editorconfig:3]: maybe we should remove the other file types so it doesn't interfere with other projects?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (steveoh) [inline .vscode/settings.json:1]: should we add the extensions.json file that recommends what plugins to also install for vscode users?

**Comment by stdavis** [comment]
@steveoh `.pylintrc` seems pretty verbose. Are those all overrides or is this something that is auto-generated? Maybe we should trim it down to just the overrides?

**Comment by stdavis** [comment]
What do you think of this plugin to enforce single quotes? https://github.com/edaniszewski/pylint-quotes

**Comment by steveoh** [comment]
> @steveoh `.pylintrc` seems pretty verbose. Are those all overrides or is this something that is auto-generated? Maybe we should trim it down to just the overrides?

I think it was generated from a CLI or something. If you are feeling brave you can generate a new one and look at the diff?

File: .editorconfig (hunk 1 above)
Comment (stdavis) [reply .editorconfig:3]: You mean just switching it to something like this?
```
[*]
indent_size = 4
```

File: .vscode/settings.json (hunk 2 above)
Comment (stdavis) [reply .vscode/settings.json:1]: ac82ddd

File: .editorconfig (hunk 1 above)
Comment (steveoh) [reply .editorconfig:3]: i was thinking more python only.

**Comment by stdavis** [comment]
I generated a new `.pylintrc` and almost every line was changed in some way. It looked like they have updated the format and defaults. So I cleared ours out other than loading the quotes plugin so that we can start fresh with overrides only.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
 # agrc/python
 
 AGRC's default Python project configuration/template
+
+## Installation
+
+1. Create new environment for project.
+1. `pip install pylint-quotes`
```
Comment (steveoh) [inline README.md:8]:
```suggestion
1. `pip install pylint pylint-quotes yapf`
```

**Pull request merged**